tests/utils/test_tokenizers.py

```diff
@@ -63,3 +66,40 @@ def test_out_of_range_ascii():
     # characters out of ascii range are mapped to zero (null)
     assert tokens[0] == 0
     assert tokens[1] == 0
+
+def test_single_token():
```

Comment:
we didn't previously have any tests using sentencepiece so the dependency will need to be added to test-requires.txt